std_msgs/Header header
int64 in_water              # in_water reading. 0 is interpreted as false
                            # it could be either 0 or 1

